include/nanobind/nb_eval.h

```diff
@@ -0,0 +1,97 @@
+/*
+    nanobind/nb_eval.h: Support for evaluating Python expressions and statements
+                        from strings and files
+
+    Adapted by Nico Schlömer from pybind11's eval.h with
+
+    Copyright (c) 2016 Klemens Morgenstern <klemens.morgenstern@ed-chemnitz.de> and
+                       Wenzel Jakob <wenzel.jakob@epfl.ch>
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+
+#pragma once
+
+#include <nanobind/nanobind.h>
+
+#include <string>
```

Comment:
Let's remove the `string` header here, we can do without.

Reply:
Sure? Because the script is using `std::string`. I'm a bit rusty on C++, but I faintly remember to "include what you use". Will go ahead with the removal once you confirm.